docs/data/date-pickers/localization/localization.md

```diff
@@ -30,9 +30,7 @@ you just have to import the locale from `@mui/x-date-pikers` (see the [list of s
 
 ```jsx
 import { createTheme, ThemeProvider } from '@mui/material/styles';
-import { DateCalendar, LocalizationProvider, deDE } from '@mui/x-date-pickers';
-import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
-import 'date-fns/locale/de';
+import { deDE } from '@mui/x-date-pickers';
```

Comment:
@flaviendelangle Should we go this route on this doc?
It had all the previous imports left after refactoring simplifying examples.
I've removed every import that is not used in the code example.

Reply:
That's fine for me